Orbital-Docs/_pages/docs/quick-start-guide/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: single
+title: Quick-Start Guide
+permalink: /docs/quick-start-guide
+sidebar:
+  nav: 'docs'
+---
+
+This quick start guide will show you how to make a simple petstore which returns a pet by name. If no pets are found, it will return a custom message.
+
+1. Follow the installation instructions to start up the designer and server
```

Comment:
is it possible to add the link to the installation page?

Reply:
Done!